### DISCUSSION TITLE
How to customise input forms

### DISCUSSION THREAD

**Discussion by KingMonty** [ory/kratos#3873] [Q&A]
I'm new to Ory Kratos, and managed to get everything working locally using [kratos-selfservice-ui-react-nextjs](https://github.com/ory/kratos-selfservice-ui-react-nextjs)

I can't figure out how to customise the form UI's in login.

I want to be able to add some fields, as well as the ability to change the text descriptions, etc. Am I correct in assuming I would need to override ory/elements to do this?

Thanks in advance!

**Comment by vinckr** [comment]
Hello @KingMonty 

Yes when you are using ory/elements you need to override the values. I think there are some pointers in the README. 
Btw we are working on the 1.0 version of Ory Elements which will make this much easier.
